app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    try:
+        mv_command, old_file, new_file = command.split()
+    except ValueError:
+        raise Exception("Enter correct command!")
+
+    if old_file != new_file and mv_command == "mv":
+        dir_path = os.path.dirname(new_file)
+        if dir_path and not os.path.exists(dir_path):
+            dirs = dir_path.split("/")
+            dir_ = ""
+            for temp in dirs:
+                dir_ += temp + "/"
```

Comment:
![image](https://github.com/mate-academy/py-move-file/assets/106102798/56eed926-840a-4c73-b835-9f91be6b4d93)
Some common mistakes are listed in checklist, I'd recommend to change this line

Reply:
I've missed that point, thank you for help!